requirements-webapp.txt

```diff
@@ -19,7 +19,7 @@ numpy
 oauthlib
 openapi-spec-validator
 orjson
-pandas
+pandas>=2
```

Comment:
For the future I recommend to have all dependencies precisely pinned, but this change has nothing to do with that idea.

This is not an important change. Just a self-documenting aspect. Background: saw 
```
/app/conbench/app/benchmarks.py:217: DeprecationWarning: parsing timezone aware datetimes is deprecated; this will raise an error in the future
```
in Arrow Conbench, could not reproduce locally. I was wondering which pandas version we use/specify. We did not specify any, i.e. in the last few months at some point we implicitly did the pandas 1.x -> 2.x migration. I just wanted to make this a tiny little more explicit, and with this change also 'forced' a rebuild of otherwise cached docker image layers.